package.json

```diff
@@ -10,7 +10,7 @@
     "test:coverage": "jest --coverage",
     "precommit": "lint-staged && npm test",
     "prepush": "npm test",
-    "build": "webpack --config webpack.prod.js",
+    "heroku-postbuild": "webpack --config webpack.prod.js",
```

Comment:
what about postinstall ?

Reply:
the start script will serve the express server that will serve the static page.